include/Lv2Worker.h

```diff
@@ -69,7 +70,7 @@ class Lv2Worker
 	std::size_t bufferSize() const;  //!< size of internal buffers
 
 	// parameters
-	const LV2_Worker_Interface* m_iface;
+	const LV2_Worker_Interface* m_interface;
```

Comment:
```suggestion
	const LV2_Worker_Interface* m_interface = nullptr;
```
Now that it's no longer being initialized by the constructor, this should be set to null just to be safe.

Reply:
Thanks, though I used this on purpose to tell the reader: We do not make any assumptions about this value before we initialize it. So I would prefer to keep this.

Reply:
As a reader, that is not what I thought when I saw it. Instead I thought, "oh, he forgot to initialize a member variable". Since it is implicitly and not explicitly uninitialized, it's something very easy to miss and can't really be called a message to the reader.

The second thing I thought was "it will be a problem if that variable is ever read from in that state".
Say in the future someone less familiar with your code (or even yourself) introduces a null pointer check to see if the interface has been set - then you'd have UB. And an unnecessary headache debugging it. So I think it's good practice to simply initialize all member variables during construction unless there's a good reason not to (such as performance), and in this case I don't see a reason not to.